restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
 import argparse
 import json
 import sys
+import locale
 
 from .lint import lint_file
 
 
-def _main(filepath, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format, encoding, stream=sys.stdout):
     # Read and lint the file
-    errors = lint_file(filepath, encoding=encoding)
+    file_errors = [lint_file(f, encoding=encoding) for f in filepaths]
+    errors_dicts = []
```

Comment:
This should be `error_dicts` as it should be a list of dictionaries. We shouldn't be pluralizing both words, only the second one.